src/view-handler/index.js

```diff
@@ -29,6 +29,7 @@ export default function createViewHandler({ viewState, extremumModel, layoutServ
         meta.isHomeState = false;
       }
       extremumModel.command.updateExtrema(dataView, meta.isHomeState);
+      meta.scale = (xMax1 - xMin1) / (xMax2 - xMin2);
```

Comment:
You changed the scale calculation, before was `const newScale = zoomFactor * scale;`. I don't think we want to change how much to zoom.